examples/transmitter/swagger_server/business_logic/generate_event.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2021 Cisco Systems, Inc. and its affiliates
+# All rights reserved.
+# Use of this source code is governed by a BSD 3-Clause License
+# that can be found in the LICENSE file.
+
+
+from swagger_server.events import (
+    SecurityEvent,
+    SessionRevoked, TokenClaimsChange, CredentialChange,
+    AssuranceLevelChange, DeviceComplianceChange,
+    AccountDisabled, AccountEnabled, AccountPurged, IdentifierChanged,
+    IdentifierRecycled, OptIn, OptOutCancelled, OptOutEffective,
+    OptOutInitiated, RISCSessionsRevoked, RecoveryActivated,
+    RecoveryInformationChanged
+)
+from swagger_server.models import (
+   Subject, EventType
+)
+
+
+event_type_map = {
+    EventType.session_revoked: SessionRevoked,
+    EventType.token_claims_change: TokenClaimsChange,
+    EventType.credential_change: CredentialChange,
+    EventType.assurance_level_change: AssuranceLevelChange,
+    EventType.device_compliance_change: DeviceComplianceChange,
+    EventType.account_purged: AccountPurged,
+    EventType.account_disabled: AccountDisabled,
+    EventType.account_enabled: AccountEnabled,
+    EventType.identifier_changed: IdentifierChanged,
+    EventType.identifier_recycled: IdentifierRecycled,
+    EventType.opt_in: OptIn,
+    EventType.opt_out_initiated: OptOutInitiated,
+    EventType.opt_out_cancelled: OptOutCancelled,
+    EventType.opt_out_effective: OptOutEffective,
+    EventType.recovery_activated: RecoveryActivated,
+    EventType.recovery_information_changed: RecoveryInformationChanged,
+    EventType.RISC_sessions_revoked: RISCSessionsRevoked,
+}
+
+
+def generate_security_event(event_type: EventType,
+                            subject: Subject) -> SecurityEvent:
+    event_class = event_type_map[event_type]
+    event_attribute_name = event_type.value.replace("-", "_")
```

Comment:
Now that `event_type` is an Enum, we can do this even more cleanly:
```suggestion
    event_attribute_name = event_type.name
```